style/styles_stories.css

```diff
@@ -0,0 +1,267 @@
+.stories {
+  padding-top: 63px;
+}
+
+.stories__title {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 22px;
+  text-align: center;
+  margin-bottom: 40px;
+}
+
+.stories__block {
+  display: flex;
+  justify-content: space-between;
+}
+
+.pic-first {
+  display: none;
+}
+
+.stories-item {
+  width: 307px;
+}
+
+.stories-item__pic {
+  padding-bottom: 29px;
+}
+
+.stories-item__title {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 18px;
+  margin: 0;
+  padding-bottom: 9px;
+}
+
+.stories-item__article {
+  font-family: "Gotham Pro Regular", sans-serif;
+  font-size: 14px;
+  line-height: 17.5px;
+  padding-bottom: 18px;
+}
+
+.stories-item__block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stories-item__link {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 12px;
+  text-decoration: none;
+  color: #000;
+}
+
+.stories-item__link::after {
+  content: url("../pic/arrow_stories.png");
+  padding-left: 7px;
+}
+
+.stories-item__date {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 12px;
+  opacity: 0.3;
+}
+
+@media (max-width: 1085px) {
+  .stories-item {
+    width: 280px;
+  }
+
+  .stories-item__pic {
+    width: 290px;
+  }
+}
+
+@media (max-width: 1024px) {
+  .stories {
+    padding-top: 78px;
+  }
+
+  .stories-item {
+    width: 277px;
+  }
+
+  .stories-item__pic {
+    width: 277px;
+    height: 147.5px;
+  }
+
+  .pic-first {
+    display: inline;
+  }
+
+  .pic-first-big {
+    display: none;
+  }
+
+  .stories-item__article {
+    font-size: 13px;
+    line-height: 16px;
+    padding-bottom: 24px;
+  }
+
+  .stories-item__link::after {
+    content: url("../pic/arrow_stories1024.png");
+    padding-left: 0;
+  }
+}
+
+@media (max-width: 940px) {
+  .stories-item {
+    width: 230px;
+  }
+
+  .stories-item__pic {
+    width: 230px;
+    height: 120px;
+  }
+}
+
+@media (max-width: 768px) {
+  .stories {
+    background-color: #000;
+    color: #fff;
+    padding-top: 26px;
+  }
+
+  .stories__title {
+    font-size: 30px;
+    margin-bottom: 42px;
+  }
+
+  .item-third {
+    display: none;
+  }
+
+  .stories__block {
+    justify-content: center;
+  }
+
+  .item-first {
+    width: 331px;
+    padding-right: 10px;
+  }
+
+  .item-second {
+    width: 312px;
+    padding-left: 28px;
+  }
+
+  .stories-item__pic {
+    width: 312px;
+    height: 166px;
+    padding-bottom: 28px;
+  }
+
+  .stories-item__title {
+    font-size: 25px;
+    padding-bottom: 8px;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)